src/node/internal/streams_util.js

```diff
@@ -867,7 +867,7 @@ eos.finished = finished;
 function checkError(err, w, r) {
   if (err) {
     // Avoid V8 leak, https://github.com/nodejs/node/pull/34103#issuecomment-652002364
-    err.stack; // eslint-disable-line no-unused-expressions
+    err.stack;
```

Comment:
Not related to this PR but just lol.

Reply:
This was throwing a unused eslint directive error